### ISSUE TITLE
🔗 🐞 Bug Report: Dark mode editor preference is effecting website design elements (Duplicate of #21809)

### ISSUE THREAD

**Issue by leeneilson** [elementor/elementor#21832]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

I have dark mode active on my laptop and since the latest update to Elementor, this is effecting elements on the website design. For example radio buttons in forms are showing with dark backgrounds, even though the rest of the website is light. This not only happens when editing a page, but also when just viewing the website frontend when logged in.

The thing that is causing this is a css file being included '/wp-content/plugins/elementor/assets/css/theme-dark.min.css?ver=3.12.0', which has the css value 'color-scheme: dark' being set on 'root'. So it effects everything, not just the editor UI.

### Steps to reproduce

1) Create a form with a radio element in it and add some values.
2) Set the Elementor user preference of 'UI Theme' to 'Dark', or to 'Auto Detect' and set your computer dark mode on.
3) View the form inside of the editor and also on the front end when saved.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MariaDB Server v10.4.28
	PHP Version: 7.4.33
	PHP Memory Limit: 128M
	PHP Max Input Vars: 2500
	PHP Max Post Size: 128M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2
	WP Multisite: No
	Max Upload Size: 128 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Language: en-GB
	Timezone: Europe/London
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor Child
	Version: 1.0
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.7.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: en_GB
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/111.0

== Active Plugins ==
	Elementor
		Version: 3.12.0
		Author: Elementor.com

	Elementor Pro
		Version: 3.12.0
		Author: Elementor.com


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	
	recaptcha_v3: Active


== Elementor Experiments ==
	Optimised DOM Output: Inactive by default
	Improved asset loading: Inactive by default
	Improved CSS Loading: Inactive by default
	Inline Font Icons: Inactive by default
	Accessibility improvements: Active by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Inactive by default
	Upgrade Swiper Library: Inactive by default
	Default to New Theme Builder: Active by default
	Editor Top Bar: Inactive by default
	Landing pages: Active by default
	Nested Elements: Inactive
	Lazy Load Background Images: Inactive by default
	Page Transitions: Active by default
	Notes: Inactive
	Loop: Active by default
	Scroll Snap: Active by default
	Menu: Inactive by default



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibility not specified

== Elementor Pro - Compatibility Tag ==

````

</details>

**Comment by markbosky** [comment]
Experiencing the same issue with "Dark Mode" affecting actual page elements. My chatbot icon now has a white container background where it should be transparent. When I toggle UI back to "light" mode the background goes back to being transparent as it should be.

**Comment by leeneilson** [comment]
@doronwolf all looks ok when not logged in or in incognito mode. So only effecting what my clients and their users see when logged in. Thanks for your help.

**Comment by markbosky** [comment]
It is only appearing when users are logged-in to WordPress with "dark mode" UI activated. It also has changed form field input backgrounds to dark. Again, only when logged-in to WP.

**Comment by nicholaszein** [comment]
Hello everyone! 

Thank you for submitting your issue! 🙏 

We are aware that the new color scheme of Elementor 3.12 is affecting some styles that it shouldn't be affecting. It seems it only happens when the user is logged in.

A fix is underway.

### 🔗 🐞 This issue is Related to #21809 

Which is on the same subject. 

For this reason, I'm closing this thread and 🔃 **Merging** it with the original, so we can have all the discussions in one location, and deal with it more efficiently.

Feel free to continue the conversation there. 😉 

Best regards

**Comment by nicholaszein** [comment]
Hello!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor **v3.12.1**! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

Cheers 🥂
